ci-operator/step-registry/cucushift/vsphere/upi/OWNERS

```diff
@@ -0,0 +1,3 @@
+approvers:
+- jhou1
+- liangxia
```

Comment:
Don't include yourself?

Reply:
not included in previous prow jobs as well. is it mandatory to add our names also?

Reply:
It would be helpful to add yourself here as a future maintainer of this configuration.